script/cibuild

```diff
@@ -4,5 +4,4 @@ sudo service dbus start
 eval $(gnome-keyring-daemon)
 export GNOME_KEYRING_CONTROL GNOME_KEYRING_PID GPG_AGENT_INFO SSH_AUTH_SOCK
 
-# FIXME Find out why we cound not connect to gnome-keyring-daemon.
-# grunt test
+grunt test
```

Comment:
Judging from the output on Travis, we're not actually running any tests... the script is actually failing, but old versions of `xvfb-run` seem to be returning hiding that fact, and always eturning with a success exit code.